packages/e2e/lib/e2e_perf.dart

```diff
@@ -55,9 +56,8 @@ Future<void> watchPerformance(
   binding.addTimingsCallback(watcher);
   await action();
   binding.removeTimingsCallback(watcher);
-  // TODO(CareF): determine if it's running on firebase and report metric online
   final FrameTimingSummarizer frameTimes = FrameTimingSummarizer(frameTimings);
-  binding.reportData = <String, dynamic>{'performance': frameTimes.summary};
+  binding.reportData = <String, dynamic>{metricName: frameTimes.summary};
```

Comment:
Why not `binding.reportData = frameTimes.summary;`?

If we have to add that 'performance' key, I'd probably call that `reportKey` instead of `metricName`. The `metricName` sounds more like `frame_rasterization_time`...

Reply:
Discussed offline. This is for allowing in the future to add more keys in the response data, without having to modify existing performance test. `metricName` => `reportKey` Done